src/pivot-table/hooks/__tests__/use-column-width.test.ts

```diff
@@ -247,10 +250,24 @@ describe("useColumnWidth", () => {
 
       dimInfo = { columnWidth: { type: ColumnWidthType.FitToContent } } as ExtendedDimensionInfo;
       visibleTopDimensionInfo = [dimInfo, -1, dimInfo];
+      layoutService.layout.qHyperCube.qEffectiveInterColumnSortOrder = [0, 1, -1, 2];
 
       const { getRightGridColumnWidth } = renderUseColumnWidth();
       expect(getRightGridColumnWidth()).toBe(width);
     });
+
+    test("should return right column width for non-pseudo dimension for fit to content when dimension is collapsed", () => {
+      const width = 40;
+      mockEstimateWidth(width);
+      mockMeasureText(width);
```

Comment:
Consider changing the return value of this to `width + 1` so that the test is guaranteed to pass only when `estimatedWidth + EXPAND_ICON_WIDTH` matches

Reply:
you mean that I mock with `width` for both of them?

Reply:
If someone was to refactor or change the code from:
```ts
        const fitToContentWidth = Math.max(
          measureTextForHeader(qFallbackTitle),
          estimateWidthForRowContent(qApprMaxGlyphCount) + iconWidth,
        );
```

to:
```ts
        const fitToContentWidth = Math.max(
          measureTextForHeader(qFallbackTitle) + iconWidth,
          estimateWidthForRowContent(qApprMaxGlyphCount),
        );
```

The test would still pass. Which may or may not be intentional.

Reply:
yeah no I don't think that should pass, especially if you change it unintentionally